CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/TeamsAPI/Models/TaskModuleEnvelope.cs

```diff
@@ -0,0 +1,37 @@
+// <auto-generated /> Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Bot.Connector.Teams.Models
+{
+    using System.Linq;
+
+    /// <summary>
+    /// Envelope for Task Module.
+    /// </summary>
+    public partial class TaskModuleEnvelope
```

Comment:
This represents the structure of the response to invoke for TaskModules. For ComposeExtensions, we used ComposeExtensionsResponse. Recommend to follow the same and rename to TaskModuleResponse.